frontend/src/routes/(authenticated)/(dashboards)/admin/+page.svelte

```diff
@@ -9,19 +9,36 @@
   import DeleteUserModal from './DeleteUserModal.svelte';
   import EditUserAccount from './EditUserAccount.svelte';
   import type { LoadAdminDashboardQuery } from '$lib/gql/types';
+  import { addNotification } from '$lib/notify/';
 
   type UserRow = LoadAdminDashboardQuery['users'][0];
 
   export let data: PageData;
   let deleteModal: DeleteUserModal;
   let formModal: EditUserAccount;
-
+  let _editing: UserRow;
   async function deleteUser(id: string): Promise<void> {
     formModal.close();
-    await deleteModal.open(id);
+    const error = await deleteModal.open(id);
+    if (error === 'none'){
+      return;
+    }
+    if (error){
+      addNotification(error, 'error');
```

Comment:
I'm not quite sure when we might use error notifications, but for now I think in forms we should keep striving to put the error in the form itself and not show an additional notification.
So, I think you can remove all the error notifications and the callback you added to `EditUserAccount` 😿.